src/java/us/kbase/mobu/initializer/ModuleInitializer.java

```diff
@@ -115,6 +115,7 @@ public void initialize(boolean example) throws Exception {
 		Map<String, Path> templateFiles = new HashMap<String, Path>();
 		templateFiles.put("module_typespec", Paths.get(moduleDir, specFile));
 		templateFiles.put("module_travis", Paths.get(moduleDir, ".travis.yml"));
+		templateFiles.put("module_tox", Paths.get(moduleDir, "tox.ini"));
```

Comment:
is this needed for java & perl? I'm not sure if it gets populated for those languages because it's in this hash, but it shouldn't.